test/teos/e2e/conftest.py

```diff
@@ -60,6 +60,31 @@ def create_txs(bitcoin_cli):
     return signed_commitment_tx, signed_penalty_tx
 
 
+@pytest.fixture()
```

Comment:
I think this can be merged with the `create_txs` fixture, but I'll fix it when merging with the polish cli PR since I have to go over it anyway.